### DISCUSSION TITLE
0.3.1?

### DISCUSSION THREAD

**Discussion by ruphy** [publiccodeyml/publiccode.yml#163] [General]
Hey there all, publiccode contributors and steering committee,

I was waiting for a few more PRs to appear before starting a new vote, but as I do not see much else coming I don't see any reason why we should keep delaying the vote on #160, and eventually plan a 0.3.1 soonish.

Thoughts?

**Comment by bfabio** [comment]
Agreed, let's go ahead with 0.3.1 :partying_face:
